lib/rules/web/admin_console/4.1/common_ui_elements.xyaml

```diff
@@ -51,7 +52,7 @@ clear_input_value:
 set_input_value:
   element:
     selector:
-      xpath: //input[contains(@id,'<input_field_id>')]
+      xpath: //input[@id='<input_field_id>']
```

Comment:
Why `//input[contains(@id,'<input_field_id>')]` don't work? using `=` is more strict, do we have impact on old scenarios using it?

Reply:
on route create page, there are elements id="name" and id="hostname", so restrict match is needed. 
No old scenario is affected.